python/ray/includes/unique_ids.pxi

```diff
@@ -129,12 +130,26 @@ cdef class UniqueID(BaseID):
 
 
 cdef class ObjectID(BaseID):
-    cdef CObjectID data
-    cdef object buffer_ref
+    cdef:
+        CObjectID data
+        object buffer_ref
+        c_bool in_core_worker
```

Comment:
Can you add a comment saying what this is? I don't really understand what this variable means still after looking at the code below.

Reply:
Is the idea that this should be False for object IDs that don't actually have an associated value?

Reply:
Added a comment.